httpbakery/form/form.go

```diff
@@ -58,27 +58,44 @@ type Filler interface {
 
 // SetUpAuth configures form authentication on c. The VisitWebPage field
 // in c will be set to a function that will attempt form based
-// authentication using h to perform the interaction with the user.
+// authentication using f to perform the interaction with the user.
```

Comment:
There seems to be some overlap between these two sentences.

Perhaps just:
     // SetUpAuth configures form authentication on c. The VisitWebPage field
     // in c will be set to a function that will attempt form-based
     // authentication using f to perform the interaction with the user
     // and fall back to using the current value of VisitWebPage
     // if form-based authentication is not supported.

?